mongodb-datastore/handlers/events.go

```diff
@@ -94,11 +96,59 @@ func SaveEvent(event *models.KeptnContextExtendedCE) error {
 		logger.Error(err.Error())
 		return err
 	}
-
 	logger.Debug(fmt.Sprintf("insertedID: %s", res.InsertedID))
+
+	// store mapping between context ID and project in separate table
+	/*
+		_, err = getProjectForContext(event.Shkeptncontext)
+		if err != nil {
+		}
+
+	*/
+	contextToProjectCollection := client.Database(mongoDBName).Collection(contextToProjectCollection)
+
+	_, err = contextToProjectCollection.InsertOne(ctx,
```

Comment:
The `contextToProjectCollection` always contains one document for one `shkeptncontext`. 
If now a document with an `shkeptncontext` is already contained in the collection, this `InsertOne` will produce an error. 
Should we check if the collection `contextToProjectCollection` contains already the `shkeptncontext` and only insert it if it is not contained?

Reply:
I will add an additional check for that